STY: Add Slack link

rstoneback opened the pull request:
# Description

Addresses # (issue)

Updates readme to include location of Slack self-invitation link on GitHub webpage. Also includes a note about development meetings.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. Please also list any relevant details for
your test configuration

- Clink the links in the readme


**Test Configuration**:
* Operating system: MacOS
* Version number: Python 3.8
* Any details about your local setup that are relevant

# Checklist:

- [ ] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

aburrell commented:
Another idea is to add this link as the project URL.  We can update that regularly (not on a release schedule).

rstoneback commented:
Keeping it outside a release schedule sounds very good. I'm not sure this is the URL info you are talking about, but I added the shared link to the website area 'about' on main pysat/pysat page.

aburrell commented:
I made the updates.  But you can fix anything by going to "settings" on the project page: https://github.com/pysat

aburrell commented on README.md:
```diff
@@ -22,6 +22,10 @@ JGR-Space Physics [Publication](https://doi.org/10.1029/2018JA025297)
 
+Come join us on Slack! An invitation to the pysat workspace is available 
+in the 'About' section of the [pysat GitHub Repository.](https://github.com/pysat/pysat)
+Development meetings are generally held biweekly.
```
```suggestion
Development meetings are generally held fortnightly.
```

aburrell reviewed (commented):
Recommend adding this to contributing, so it gets into the docs.  Also fine to have it on the README.

rstoneback commented:
> I made the updates. But you can fix anything by going to "settings" on the project page: https://github.com/pysat

Thanks! I had never seen that area before despite being on that page countless times. I updated the twitter handle to the latest, pysatDevelopers. I've also updated the email to `pysatDevelopers@gmail.com` for consistency.

rstoneback commented:
Thanks @aburrell!
